### DISCUSSION TITLE
Map q-tree with data from vuex store

### DISCUSSION THREAD

**Discussion by zorlakov** [quasarframework/quasar#10267] [General - Components / Directives / etc] [answered]
I have a simple store module with this data: 
const state = () => ({
  filterActivities: [],
});
with a getter and mutation to set the value of the state to the response of an API get request which works. I have been struggling how to send this data to the quasar tree and map it successfully, so far it's just been errors. 
Currently in setup I have:
    const someData = ref([]);

in computed I have:
 ...mapGetters({
          someData: 'getAllFilters',
        }),

but it doesn't work. Is there something I'm missing with q-tree and vuex store?

**Comment by metalsadman** [comment, marked as answer]
@zorlakov map helpers seem to only work with options api, here's how to use getters in composition api https://next.vuex.vuejs.org/guide/composition-api.html#accessing-state-and-getters.
